### PR TITLE
feat: add V2 projections 

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+{
+    using System;
+    using Infrastructure;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+    public class ParcelExtractItemV2
+    {
+        public Guid? ParcelId { get; set; }
+        public string? CaPaKey { get; set; }
```
Comment (ArneD) [inline src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2.cs:11]: Aren't these required?

File: src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2Projections.cs [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    using Parcel.Events;
+    using Microsoft.Extensions.Options;
+    using NodaTime;
+    using Parcel;
+
+    [ConnectedProjectionName("Extract percelen")]
+    [ConnectedProjectionDescription("Projectie die de percelen data voor het percelen extract voorziet.")]
+    public class ParcelExtractV2Projections : ConnectedProjection<ExtractContext>
+    {
+        private const string InUse = "Gerealiseerd";
```
Comment (ArneD) [inline src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2Projections.cs:20]: Maybe rename to `Realized`

**Review by ArneD — CHANGES_REQUESTED** [review]
Legacy projection tests?

File: src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2.cs (hunk 1 above)
Comment (jvandaal) [reply src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2.cs]: They are.

File: src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2Projections.cs (hunk 2 above)
Comment (jvandaal) [reply src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractV2Projections.cs]: Changed

File: src/ParcelRegistry.Projections.Legacy/LegacyContext.cs [hunk 3]
```diff
@@ -4,18 +4,26 @@ namespace ParcelRegistry.Projections.Legacy
     using Infrastructure;
     using Microsoft.EntityFrameworkCore;
     using ParcelDetail;
+    using ParcelDetailV2;
     using ParcelSyndication;
+    using ParcelDetailListViewCount = ParcelDetail.ParcelDetailListViewCount;
```
[the remark this answers is not in the document]
Comment (jvandaal) [reply src/ParcelRegistry.Projections.Legacy/LegacyContext.cs]: Removed.

File: src/ParcelRegistry.Projector/Infrastructure/Startup.cs [hunk 4]
```diff
@@ -94,31 +93,32 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
...
+                            //health.AddDbContextCheck<LegacyContext>(
+                            //    $"dbcontext-{nameof(LegacyContext).ToLowerInvariant()}",
+                            //    tags: new[] {DatabaseTag, "sql", "sqlserver"});
 
-                            health.AddDbContextCheck<LastChangedListContext>(
-                                $"dbcontext-{nameof(LastChangedListContext).ToLowerInvariant()}",
-                                tags: new[] {DatabaseTag, "sql", "sqlserver"});
+                            //health.AddDbContextCheck<LastChangedListContext>(
+                            //    $"dbcontext-{nameof(LastChangedListContext).ToLowerInvariant()}",
+                            //    tags: new[] {DatabaseTag, "sql", "sqlserver"});
```
[the remark this answers is not in the document]
Comment (jvandaal) [reply src/ParcelRegistry.Projector/Infrastructure/Startup.cs]: Undone

**Review by ArneD — APPROVED** [review]

**Pull request merged**